src/class.js

```diff
@@ -1,24 +1,31 @@
-// Create a class called User.
-// The constructor of the class should have a parameter called `options`.
-// `options` will be an object that will have the properties `email` and `password`.
-// Set the `email` and `password` properties on the class.
-// Add a method called `comparePasswords`.  `comparePasswords` should have a parameter
-// for a potential password that will be compared to the `password` property.
-// Return true if the potential password matches the `password` property.  Otherwise return false.
+class User {
+  constructor(options) {
+    this.email = options.email;
+    this.password = options.password;
+  }
+  comparePasswords(potentialPassword) {
+    return this.password === potentialPassword;
+  }
+}
 
+class Animal {
+  constructor(options) {
+    this.age = options.age;
+  }
+  growOlder() {
+    return this.age + 1;
```

Comment:
This should be:
```
this.age++;
return this.age;
```
Right now your method will return 2 every single time it is called